sunpy/data/sunpyrc

```diff
@@ -51,6 +51,37 @@ sample_dir = data/sample_data
 ; Location to specify sunpy database parameters.
 ; The default url of the database location can specified here. The url of
 ; the database comprises the database driver, its location and name, as well
-; as optional authentication parameters 
+; as optional authentication parameters
 ; Default value: sqlite:////<user's home directory>/sunpy/sunpydb.sqlite
-; url = sqlite:////home/$USER/sunpy/sunpydb.sqlite
\ No newline at end of file
+; url = sqlite:////home/$USER/sunpy/sunpydb.sqlite
+
+;;;;;;;;;;;;
+; Logger   ;
+;;;;;;;;;;;;
+[logger]
+
+# Threshold for the logging messages. Logging messages that are less severe
+# than this level will be ignored. The levels are 'DEBUG', 'INFO', 'WARNING',
+# 'ERROR'
+log_level = INFO
+
+# Whether to use color for the level names
+use_color = True
+
+# Whether to log warnings.warn calls
+log_warnings = True
+
+# Whether to log exceptions before raising them
+log_exceptions = False
+
+# Whether to always log messages to a log file
+log_to_file = False
+
+# The file to log messages to
+log_file_path = sunpy.log
```

Comment:
I think we shoudn't define this in the template config, hence the suggestion of adding a comment infront, we want the user to explicitly set where they want to save this.
```suggestion
# log_file_path = sunpy.log
```